dalme_app/static/css/dalme_public/dalme_public.css

```diff
@@ -2819,7 +2818,7 @@ section.transcription {
   background-color: #fff;
   flex: 1 1 auto;
   z-index: 1;
-  min-height: calc(100vh - (var(--header-height) + 150px));
+  height: calc(100vh - (var(--header-height) + 88px));
```

Comment:
Should this be `height: calc(100vh - (var(--header-height) + 96px));`? I'm counting as 54px from the footer + 42px from the toolbar.

`section.transcription` also needs a `min-height: 100px` to match a similar constraint in the default diva-outer class.